grand_challenge_forge/partials/docker-bash-scripts/do_build.sh.j2

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Stop at first error
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+DOCKER_IMAGE_TAG="{{ image_tag }}"
+
+
+# Check if an argument is provided
+if [ "$#" -eq 1 ]; then
+    DOCKER_IMAGE_TAG="$1"
+fi
+
+# Note: the build-arg is JUST for the workshop
+docker build "$SCRIPT_DIR" \
+  --platform=linux/amd64 \
+  --build-arg BUILD_VERSION=$(date +%s) \
+  --tag "$DOCKER_IMAGE_TAG" 2>&1
```

Comment:
I think you want to leave the build arg out of this, right?

Reply:
Ugh! Yes! I neglected to remove that. Good catch!